### DISCUSSION TITLE
exporting output KG to specific directory and file

### DISCUSSION THREAD

**Discussion by verarmaria** [morph-kgc/morph-kgc#133] [Q&A] [answered]
Hi,

I'm having troubles with exporting the resulting knowledge graph.

As far as I understood it is configured in the "config". Even though I clarify the output file name and the output dir, it doesn't create the file.

My config is:

    config = """ 
                 
                 [CONFIGURATION] 
                 output_file: knowledge-graph.nt
                 output_dir: C:/Users/Documents/venv
                 
                 [DataSource1]
                 mappings: C:/Users/Documents/venv/mapping.ttl
                
    """

is there anything I'm overlooking?

**Comment by arenas-guerrero-julian** [comment, marked as answer]
Hi @verarmaria ,

The documentation says the following about `output_dir`: _Directory to write the resulting knowledge graph to. If it is specified, `output_file` will be ignored and multiple output files will generated_

So if you want to output the resulting knowledge graph to only one file, you should remove `output_dir` in your config.

Also keep in mind that the output knowledge graph is only written to a file if executing via comand line.
